utils/protobuf/rxsamplesmetadata.proto

```diff
@@ -0,0 +1,10 @@
+syntax = "proto3";
```

Comment:
Was the computationpacket.proto file deleted in this pull request or another one?